psi4/src/psi4/dfocc/get_moinfo.cc

```diff
@@ -339,35 +339,10 @@ void DFOCC::get_moinfo() {
     /********************************************************************************************/
     /************************** Create all required matrice *************************************/
     /********************************************************************************************/
-    // Build Hso
-    Hso_ = std::shared_ptr<Matrix>(new Matrix("SO-basis One-electron Ints", nso_, nso_));
-    Tso_ = std::shared_ptr<Matrix>(new Matrix("SO-basis Kinetic Energy Ints", nso_, nso_));
-    Vso_ = std::shared_ptr<Matrix>(new Matrix("SO-basis Potential Energy Ints", nso_, nso_));
-    Sso_ = std::shared_ptr<Matrix>(new Matrix("SO-basis Overlap Ints", nso_, nso_));
-    Hso_->zero();
-    Tso_->zero();
-    Vso_->zero();
-    Sso_->zero();
-
-    // Read SO-basis one-electron integrals
-    double *so_ints = init_array(ntri_so);
-    IWL::read_one(psio_.get(), PSIF_OEI, PSIF_SO_T, so_ints, ntri_so, 0, 0, "outfile");
-    Tso_->set(so_ints);
-    IWL::read_one(psio_.get(), PSIF_OEI, PSIF_SO_V, so_ints, ntri_so, 0, 0, "outfile");
-    Vso_->set(so_ints);
-    IWL::read_one(psio_.get(), PSIF_OEI, PSIF_SO_S, so_ints, ntri_so, 0, 0, "outfile");
-    Sso_->set(so_ints);
-    free(so_ints);
-    Hso_->copy(Tso_);
-    Hso_->add(Vso_);
-    Tso_.reset();
-    Vso_.reset();
     Hso = SharedTensor2d(new Tensor2d("SO-basis One-electron Ints", nso_, nso_));
```

Comment:
FYI, @CDSherrill, I think they're out to change your dfocc OEI yet again.

Reply:
Yup, centralizing it will allow various terms like ECPs to work transparently.  IF you know of any other offending lines of code stuck in T+V world, please let me know.